commands/memberinfo.js

```diff
@@ -41,8 +41,15 @@ module.exports = {
 		if (((profile_data.xpTimeoutUntil - message.createdTimestamp > 0) && (!configData.xp.xpTimeoutHidden)) || (override)) {
 			fields.push({ name: "XP Timeout", value: functions.msToString(profile_data.xpTimeoutUntil - message.createdTimestamp), inline: true });
 		}
-		let xpPercentage = Math.round(profile_data.xp / Math.pow(profile_data.level + configData.xp.levelBaseOffset, configData.xp.levelExponent) * 100);
+		
+		let xpPercentage;
+		if (profileData.xp < 0) {
+			xpPercentage = 0;
```

Comment:
Shouldn't the XP be set to 0 as well? Since negative XP values don't make any real sense??

Reply:
I don't think so, because users are able to bleed xp if they are inactive (which I think was the problem in the first place). Should they be able to bleed past 0 xp points, do you think?

Reply:
Oh! Yea, my bad! I forgot that users were to keep their levels in case they started bleeding XP... but then could we not just make it so the XP can't bleed below 0? 
To me it doesn't really make sense that you can have -100xp but still being shown 0% as your level score.

Reply:
I think we need to discuss more about that. Maybe we could have some indicator showing up in me/memberinfo command if they have been bleeding xp below 0. Or, as you suggested, setting a bleed limit at 0.

Reply:
Yep, I personally think a bleed limit at 0 would be the most intuitive and ideal option, but we can take that up with the other moderators and admins.